Shortly.Codes.Application/Exceptions/CodeIsNotValidException.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Net;
+
+namespace Shortly.Codes.Application.Exceptions;
+
+public class CodeIsNotValidException : HttpRequestException
+{
+    public const string MESSAGE = "error.code.is.not.valid";
```

Comment:
`type.assembly/module.payload`
```suggestion
    public const string MESSAGE = "error.code.invalid";
```